src/concurrency/async-pitfalls/async-traits.md

```diff
@@ -4,22 +4,20 @@ minutes: 5
 
 # Async Traits
 
-Async methods in traits are were stabilized only recently, in the 1.75 release.
-This required support for using return-position `impl Trait` (RPIT) in traits,
-as the desugaring for `async fn` includes `-> impl Future<Output = ...>`.
+Async methods in traits are were stabilized in the 1.75 release. This required
```

Comment:
Maybe we should fix "are were" while we're at it.

Reply:
I assume you mean
```suggestion
Async methods in traits are, were, and always will be stabilized in the 1.75 release. This required
```
;)